package.json

```diff
@@ -23,54 +23,62 @@
     "url": "git+https://github.com/Microsoft/PowerBI-visuals-sunburst.git"
   },
   "devDependencies": {
-    "@types/d3-hierarchy": "^3.1.2",
-    "@types/d3-selection": "^3.0.5",
-    "@types/d3-shape": "^3.1.1",
-    "@types/jasmine": "^4.3.5",
-    "@types/jasmine-jquery": "^1.5.34",
-    "@types/jquery": "3.5.16",
-    "@types/karma": "6.3.4",
-    "@types/lodash": "4.14.196",
-    "@types/webpack": "5.28.1",
+    "@types/d3-hierarchy": "^2.0.0",
+    "@types/d3-selection": "^2.0.0",
+    "@types/d3-shape": "^2.0.0",
+    "@types/jasmine": "^3.6.2",
+    "@types/jasmine-jquery": "^1.5.33",
+    "@types/jquery": "3.5.5",
+    "@types/karma": "^6.3.4",
+    "@types/lodash": "4.14.166",
+    "@types/webpack": "4.41.25",
     "@typescript-eslint/eslint-plugin": "^6.2.1",
-    "coveralls": "3.1.1",
+    "core-js": "^3.32.0",
+    "coveralls": "^3.1.0",
+    "css-loader": "^5.0.1",
     "d3-hierarchy": "^3.1.2",
     "d3-selection": "^3.0.0",
     "d3-shape": "^3.2.0",
+    "eslint": "^8.43.0",
     "eslint-plugin-powerbi-visuals": "^0.8.1",
-    "jasmine": "5.1.0",
+    "jasmine": "^5.1.0",
     "jasmine-core": "^5.1.0",
+    "jasmine-jquery": "^2.1.1",
+    "jquery": "^3.7.0",
     "karma": "^6.4.2",
     "karma-chrome-launcher": "^3.2.0",
-    "karma-coverage": "2.2.1",
+    "karma-coverage": "^2.2.1",
     "karma-coverage-istanbul-reporter": "^3.0.3",
-    "karma-jasmine": "5.1.0",
+    "karma-jasmine": "^5.1.0",
     "karma-junit-reporter": "^2.0.1",
     "karma-remap-istanbul": "^0.0.2",
-    "karma-sourcemap-loader": "0.4.0",
-    "karma-typescript": "5.5.4",
-    "karma-typescript-preprocessor": "0.4.0",
-    "karma-webpack": "5.0.0",
-    "lodash-es": "4.17.21",
-    "playwright": "^1.36.2",
+    "karma-sourcemap-loader": "^0.3.8",
+    "karma-typescript": "^5.2.0",
+    "karma-typescript-preprocessor": "^0.4.0",
+    "karma-webpack": "^5.0.0",
+    "less": "^4.0.0",
+    "less-loader": "^7.2.1",
+    "lodash-es": "^4.17.20",
     "postcss-less": "^6.0.0",
-    "powerbi-models": "^1.12.6",
+    "powerbi-models": "^1.7.0",
     "powerbi-visuals-api": "~5.4.0",
-    "powerbi-visuals-tools": "5.0.2",
+    "powerbi-visuals-tools": "^5.0.1",
```

Comment:
Please, update to 5.0.2

Reply:
It was already 5.0.2 due to the "^" but I have updated the number explicitly as well: [81189d4](https://github.com/microsoft/powerbi-visuals-sunburst/pull/95/commits/81189d443a2bda4f576855676a9396fd4aace44a)